### PR TITLE
des-1934: customized splash screen

### PR DESCRIPTION
**Pull request by lneir**
Please review, code here from GS adds customized splash screen capability.

### CODE REVIEW

File: minuet/symphony/Paragon/App.xaml.cs [hunk 1]
```diff
@@ -45,26 +45,11 @@ protected override void OnStartup(StartupEventArgs e)
...
-                            finally
-                            {
-                                Win32Api.DeleteObject(hbmp);
-                            }
-                        }
-                    }
 
-                    _splash = new ParagonSplashScreen(_appPackage.Manifest.Name, imageSource, _appPackage.Manifest.Version);
+                    var stylePart = !string.IsNullOrEmpty(_appPackage.Manifest.SplashScreenStyle) ? _appPackage.GetPart(_appPackage.Manifest.SplashScreenStyle) : null;
+                    var styleStream = stylePart != null ? stylePart.GetStream() : null;
```
Comment (jonmclachlan) [inline minuet/symphony/Paragon/App.xaml.cs:50]: Nice null checking.

Comment (vsadda) [inline minuet/symphony/Paragon/App.xaml.cs:50]: Thanks.

File: minuet/symphony/Paragon/Paragon.csproj [hunk 2]
```diff
@@ -83,6 +83,10 @@
       <SubType>Designer</SubType>
       <Generator>MSBuild:Compile</Generator>
     </Page>
+    <Page Include="ParagonSplashScreenStyle.xaml">
+      <SubType>Designer</SubType>
+      <Generator>MSBuild:Compile</Generator>
```
Comment (jonmclachlan) [inline minuet/symphony/Paragon/Paragon.csproj:88]: Hmmm.  In the other compilation XML command, it looks like we copy the style.xaml thing into the output directory.  Do we want to do the same thing here with this style.xaml too?  Or, maybe it was intentionally omitted from the output dir?

Comment (vsadda) [inline minuet/symphony/Paragon/Paragon.csproj:88]: In the Symphony project the, the build action for the style file should be “Copy always”

Comment (vsadda) [inline minuet/symphony/Paragon/Paragon.csproj:88]: In paragon project the style file should be compiled and in the symphony project the style file should be copied.

Comment (jonmclachlan) [inline minuet/symphony/Paragon/Paragon.csproj:88]: Ok, no problem.  So it is intended.  Cool.

**Pull request merged**